aspnetcore/razor-pages/ui-class.md

```diff
@@ -197,3 +197,22 @@ When a view, partial view, or Razor Page is found in both the web app and the Ra
 In the sample download, rename *WebApp1/Areas/MyFeature2* to *WebApp1/Areas/MyFeature* to test precedence.
 
 Copy the *RazorUIClassLib/Areas/MyFeature/Pages/Shared/_Message.cshtml* partial view to *WebApp1/Areas/MyFeature/Pages/Shared/_Message.cshtml*. Update the markup to indicate the new location. Build and run the app to verify the app's version of the partial is being used.
+
+<a name="afs"></a>
```

Comment:
What does this do?

Reply:
@pranavkm  that's the bookmark link for 
See  `[RCL Pages layout](#afs)` to create a RCL that exposes content in `~/Pages` rather than `~/Areas/Pages`